CMakeLists.txt

```diff
@@ -685,11 +685,12 @@ set(openPMD_TEST_NAMES
 # command line tools
 set(openPMD_CLI_TOOL_NAMES
     ls
+    convert-json-toml
```

Comment:
Also add this to `setup.py` -> `entry_points` -> `console_scripts`?

Reply:
This is not a Python module, but written in C++, so I doubt that will work?